### ISSUE TITLE
Expansion Items

### ISSUE THREAD

**Issue by Seesam24601** [Seesam24601/TAMMI#8]
Provide a way for users to mark expansion items. If possible, any specific information about how expansion items are prioritized should be handled by the user-defined priorities function.

**Comment by Seesam24601** [comment]
Blocked by #3

**Comment by Seesam24601** [comment]
This should be able to be implemented easily by the end-user by adding a column to the `assets` table and then implementing that change in a priorities function. This is not necessary for completing the core of the model or for a reasonable demonstration. Thus, I am closing this issue.

Providing built-in support for features like this may be re-added in a later phase of the project.
